app/results.py

```diff
@@ -96,11 +96,12 @@ def search(self, query_dict):
                 results = indexing.search(self.repo,
                                           TextQuery(query_txt), MAX_RESULTS)
 
+        query_name = self.params.query_name or self.params.query
```

Comment:
Can you document the reason to accept both parameter names here too? Same for below.

Or, may be better to create a helper function which returns:
`self.params.query_name or self.params.query`
instead of writing the same logic 3 times? Then you can also write the document only in the function.